packages/client/src/index.test.ts

```diff
@@ -34,19 +34,18 @@ describe('init client', () => {
   beforeAll((done) => {
     client = new LogtoClient(
       {
-        discoveryUrl: 'https://logto.dev/oidc/.well-known/openid-configuration',
+        logtoUrl: 'https://logto.dev',
         clientId: 'foo',
-        redirectUris: ['http://localhost:3000/callback'],
       },
       done
     );
   });
   test('openid configuration', async () => {
-    const configuration = client.issuer.metadata;
-    expect(configuration.authorization_endpoint).toContain('oidc/auth');
+    const configuration = client.issuer?.metadata;
+    expect(configuration?.authorization_endpoint).toContain('oidc/auth');
   });
   test('get login url and codeVerifier', () => {
-    const [url, codeVerifier] = client.getLoginUrlAndCodeVerifier();
+    const [url, codeVerifier] = client.getLoginUrlAndCodeVerifier('http://localhost:3000/callback');
     console.log(url);
```

Comment:
should we remove the console.log statement?

Reply:
It is used for manual test, automatic browser test is not implemented for now.

Reply:
looks like it's not about manual/auto test? should expect the url starts with the same host and `codeVerifier` has the correct format